src/tracer/MOM_neutral_diffusion.F90

```diff
@@ -1102,7 +1472,7 @@ logical function neutral_diffusion_unit_tests(verbose)
   neutral_diffusion_unit_tests = neutral_diffusion_unit_tests .or. test_ifndp(v,-2.0, .5,  5.0, 0.5, 0.5, 'Check dP=0')
 
   ! Identical columns
-  call find_neutral_surface_positions(3, &
+  call find_neutral_surface_positions_continuous(3, &
```

Comment:
As you noted some time before, the discontinuous reconstructions don't always lend themselves nicely to whole number results. That being said, it's on my to-do list before submitting an actual PR.